integration_tests/test_store_history.py

```diff
@@ -25,53 +25,53 @@
 import integration_tests
 
 
-class HistoryTestCase(integration_tests.StoreTestCase):
+class RevisionsTestCase(integration_tests.StoreTestCase):
```

Comment:
I added a single test method since both are handled by the same logic in the source tree.  Is that sufficient or would you prefer an entire copy of the RevisionsTestCase class?